net/tac_plus/Makefile

```diff
@@ -0,0 +1,50 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=tac_plus
+PKG_VERSION:=4.0.4.28
+PKG_REV:=f28244cdc114ef88e62e2f754dc12b58bcfaa040
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_REV).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/facebook/tac_plus/tar.gz/$(PKG_REV)?
+PKG_SOURCE_VERSION:=$(PKG_REV)
+PKG_HASH:=126a9eab59247cd7da5ece504ea23ce5ba7e20339de74e164d8e1b79e080d92f
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+PKG_BUILD_SUBDIR:=tacacs-F$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+
+PKG_UNPACK:=$(HOST_TAR) -C $(PKG_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
+
+MAKE_PATH:=$(PKG_BUILD_SUBDIR)
+CONFIGURE_PATH:=$(PKG_BUILD_SUBDIR)
+
+CONFIGURE_ARGS += \
+	--disable-shared \
+	--without-libwrap
+
+define Package/tac_plus
+	SECTION:=net
+	CATEGORY:=Network
+	TITLE:=TACACS+ daemon
+	URL:=https://www.shrubbery.net/tac_plus/
```

Comment:
As you're using facebook's fork, this is probably not a correct URL.

Reply:
That's fine, facebook doesn't have one

Reply:
Right, but normally the github repo is linked.